third_party/maya/lib/usdMaya/writeUtil.cpp

```diff
@@ -942,20 +943,22 @@ PxrUsdMayaWriteUtil::WriteUserExportedAttributes(
             if (primvar) {
                 usdAttr = primvar.GetAttr();
             }
-        } else if (usdAttrType ==
-                    PxrUsdMayaUserTaggedAttributeTokens->USDAttrTypeUsdRi) {
-            usdAttr =
-                PxrUsdMayaWriteUtil::GetOrCreateUsdRiAttribute(attrPlug,
-                                                               usdPrim,
-                                                               usdAttrName,
-                                                               "user",
-                                                               translateMayaDoubleToUsdSinglePrecision);
         } else {
-            usdAttr = PxrUsdMayaWriteUtil::GetOrCreateUsdAttr(attrPlug,
-                                                              usdPrim,
-                                                              usdAttrName,
-                                                              true,
-                                                              translateMayaDoubleToUsdSinglePrecision);
+            auto attributeWriter = PxrUsdMayaUserAttributeWriterRegistry::GetWriter(usdAttrType);
```

Comment:
Do we need to add a plugin to handle the UsdRi attribute support that was removed here? In local testing, it looks like testUsdMayaUserExportedAttributes is failing with this change, possibly for this reason.

Reply:
I think this was a bad merge on our side (or we pushed up an older version of PR), I'm looking at the latest updated code and UsdRi is there.
I updated the PR (and addressed the rest of the review items too), that should have the old piece of code back.